source/image-handler/terraform/cross_account_access.tf

```diff
@@ -0,0 +1,54 @@
+locals {
+  # idea here is to provide each team with its own distinct role
+  # to allow them publishing static assets under their team directory, e.g. /s/dcp/
+  teams = {
+    "newbiz-product-images" = {
+      account_id = 786771379108
+    }
+  }
+}
+
+resource "aws_iam_role" "s3_org_access" {
+  for_each = local.teams
+
+  name = "s3_images_access_team_${each.key}"
```

Comment:
```suggestion
  name = "s3-images-access-team-${each.key}-${data.aws_region.name}"
```

Reply:
Is there a max length for the role names? Could be tight with "newbiz-product-images"